Makefile

```diff
@@ -14,12 +14,14 @@ CMAKE_EXTRA_FLAGS ?=
 # CMAKE_INSTALL_PREFIX
 #   - May be passed directly or as part of CMAKE_EXTRA_FLAGS.
 #   - `checkprefix` target checks that it matches the CMake-cached value. #9615
+ifneq (,$(CMAKE_INSTALL_PREFIX))
 ifneq (,$(CMAKE_EXTRA_FLAGS))
```

Comment:
can you just remove these two lines?

Reply:
Would not help really.

With `CMAKE_EXTRA_FLAGS += "-DCMAKE_INSTALL_PREFIX=/home/user/.local/opt/neovim"` in local.mk "make install" fails.

This is due to `?=` making it a dynmaic variable, i.e. the shell command might even be executed multiple times.

Therefore we should only do this if `CMAKE_INSTALL_PREFIX` is not provided already.

Reply:
even if it doesn't help, those guards are just noise.

Reply:
They prevent the error in the first place.